fs_types.go

```diff
@@ -1,6 +1,6 @@
 /*
  *
- * Copyright 2020 Dell EMC Corporation
+ * Copyright © 2020-2022 Dell EMC Corporation
```

Comment:
Should it be Dell Inc. or its subsidiaries. All Rights Reserved instead of Dell EMC Corporation?

Reply:
done